src/marketplace/MakeAComponentSaleView.js

```diff
@@ -1,33 +1,35 @@
 //@flow strict
 import * as React from 'react';
 import Cookies from 'universal-cookie';
+import { ToastContainer ,toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.min.css';
 
 type Props = {||}; 
 type State = {|
-    userId: string,
-    salePrice: number,
-    itemID: string,
-    itemAmount: number,
+	userId: string,
+	salePrice: number,
+	itemID: string,
+	itemAmount: number,
 |};
 let itemsToSell = [{value: '', label: ''}];
```

Comment:
This is why you are having to do force update, so this is a no-no

Reply:
Yeah this is just a toasty review these changes are coming when I handle the infinite request which I grouped together with issue 294

Reply:
cool